src/unstructured_client/utils/_human_utils.py

```diff
@@ -88,3 +91,13 @@ def wrapper(*args: _P.args, **kwargs: _P.kwargs) -> operations.PartitionResponse
             return func(*args, **kwargs)
 
     return wrapper
+
+
+def log_retries(retry_count: int, sleep: float):
+    """Function for logging retries to give users visibility into requests."""
+    logging.basicConfig(level=logging.INFO, format='%(levelname)s: %(message)s', stream=sys.stdout)
+    logger = logging.getLogger('unstructured-client')
+    logger.setLevel(logging.INFO)
+    logger.info(
+        "Retry attempt #%s. Sleeping %s seconds before retry.", retry_count, round(sleep, 1)
```

Comment:
Any chance we can access the response here? Would be great to also dump the status code and any error message that came back.